doc/source/cluster/kubernetes.rst

```diff
@@ -1,271 +1,464 @@
-.. _ray-k8s-deploy:
-
+***********************
 Deploying on Kubernetes
-=======================
+***********************
+
+.. toctree::
+    :hidden:
+
+    /cluster/kubernetes-manual.rst
+    /cluster/kubernetes-gpu.rst
```

Comment:
can you just orphan these two files? They mess with the sidebar (on the left side)